cognite_toolkit/_cdf_tk/client/_toolkit_client.py

```diff
@@ -76,6 +79,8 @@ def __init__(self, config: ToolkitClientConfig | None = None) -> None:
         self.dml = DMLAPI(self._config, self._API_VERSION, self)
         self.verify = VerifyAPI(self._config, self._API_VERSION, self)
         self.lookup = LookUpGroup(self._config, self._API_VERSION, self)
+        if FeatureFlag.is_enabled(Flags.AGENTS):
```

Comment:
Personal preference: No need for the feature flag here. This is internal so will not be exposed to the end user anyways. You already remove the loader if the flag is not enabled.